packages/mdc-tab/_mixins.scss

```diff
@@ -266,20 +278,15 @@
   $feat-structure: mdc-feature-create-target($query, structure);
 
   @include mdc-feature-targets($feat-structure) {
-    height: $mdc-tab-stacked-height;
-
     .mdc-tab__content {
       flex-direction: column;
       align-items: center;
-      justify-content: space-between;
-    }
-
-    .mdc-tab__icon {
-      padding-top: 12px;
+      justify-content: center;
     }
 
     .mdc-tab__text-label {
-      padding-bottom: 16px;
+      padding-top: 6px;
```

Comment:
These lines look like they have potential to introduce screenshot changes in Google.

Reply:
Possibly unlikely, though. I don't believe the stacked variant is in the GM spec.

Reply:
Just ran tests. This is only affecting IE11 (Actually fixing the alignment on IE11 compared to previous screenshots).